libpod/network/create.go

```diff
@@ -155,6 +164,11 @@ func createBridge(r *libpod.Runtime, name string, options entities.NetworkCreate
 		ipMasq = false
 	}
 
+	mtu, err := parseMTU(options.Options["mtu"])
+	if err != nil {
+		return "", err
+	}
+
```

Comment:
We should be parsing all options and return and error for unknown ones. `podman network create -o bogus` should not work.

Reply:
That would make a little more sensitive to version skew, but is probably the right thing

Reply:
This would also require podman to get all the available options for all the available drivers

Reply:
I'm thinking of something similar to this:
```
for k,val := range options.Options {
    switch k {
        case "mtu":
            parsemtu(val)
        default:
            return errors.Errorf("unsupported option %s",k)
    }
}
```
It's easy for someone to have a typo. We should error right away instead of having the user to debug why his config is not working as expected.

Reply:
But these are plugins, right ? So it could be extended ?

Reply:
We could do it for the bridge plugin like indicated